tsconfig.strictTypeCheck.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This is still in the wrong location... this file should be inside the `modules/tinymce` directory since it only applies to that. I'd also suggest we just shorten the name to `tsconfig.strict.json`.